client/src/pages/Login.js

```diff
@@ -0,0 +1,56 @@
+import logoBike from "./logoBike.svg";
+import Form from "react-bootstrap/Form";
+
+
+const Login = () => {
+	return (
+		<form>
+			<div className="Login-page">
+				<div className="flex-container">
+					<div className="img">
+						<div className="container-image">
+							<img src={logoBike} alt="bike logo" />
+						</div>
+
+						<h4>sign in</h4>
+					</div>
```

Comment:
```suggestion
			<img src={logoBike} alt="bike logo" className="container-image" />

			<h1 className="fs-5">Sign in</h1>
```
The `container-image` class works on the `img` element so we can move the `container-image` class to the `img` element and delete the `div`.

The `img` class isn't necessary because our flex css is centering the logo, so we can delete the `<div className="img">` line.

The `sign in` heading can be an `h1` because it will be the main heading of the page. We can make the text size smaller by using the [bootstrap utility class](https://getbootstrap.com/docs/5.0/utilities/text/#font-size) `fs-5`. We should also capitalise `sign in` because text on the UI is usually capitalised.